requirements.txt

```diff
@@ -56,3 +56,6 @@ pyxlsb>=1.0.8       # excel
 pyarrow             # parquet
 lxml                # html
 html5lib            # html
+
+# Allows users to upload
+predibase>=2023.10.2
```

Comment:
Please move this to `requirements_extra.txt` instead.

Reply:
Done! @justinxzhao and I talked about this on Friday. Since HF is part of the core requirements, it felt like it could go either way but I agree this keeps Ludwig cleaner.